src/Controllers/DatabaseController.js

```diff
@@ -831,13 +849,19 @@ class DatabaseController {
           .then(schema => {
             transformAuthData(className, object, schema);
             flattenUpdateOperatorsForCreate(object);
+            if (validateOnly) {
+              return object;
```

Comment:
what is getting returned here?

Reply:
The original object that would go to the database but in the case of validateOnly, it is just returned after the validations.